### ISSUE TITLE
selectionQC: Fails on LDLR R2 dataset

### ISSUE THREAD

**Issue by jweile** [jweile/tileseqMave#11]
```2020-06-15_20:15:41 INFO: Reading count data 
2020-06-15_20:15:41 INFO: Processing condition lowest ; time 1 
2020-06-15_20:15:43 INFO: Processing condition lowmid ; time 1 
2020-06-15_20:15:45 INFO: Processing condition mid ; time 1 
Error in `[.data.frame`(marginalCounts, , repMatrix["selWT", repi]) : 
  undefined columns selected
Calls: selectionQC ... FUN -> floor0 -> sapply -> lapply -> [ -> [.data.frame
In addition: Warning messages:
1: In xy.coords(x, y, xlabel, ylabel, log) :
  180 x values <= 0 omitted from logarithmic plot
2: In xy.coords(x, y, xlabel, ylabel, log) :
  192 y values <= 0 omitted from logarithmic plot
3: In xy.coords(x, y, xlabel, ylabel, log) :
  207 x values <= 0 omitted from logarithmic plot
4: In xy.coords(x, y, xlabel, ylabel, log) :
  258 y values <= 0 omitted from logarithmic plot
2020-06-15_20:15:48 FATAL: Error in `[.data.frame`(marginalCounts, , repMatrix["selWT", repi]) : 
  undefined columns selected
Calls: selectionQC ... FUN -> floor0 -> sapply -> lapply -> [ -> [.data.frame
```

**Comment by jweile** [comment]
This is caused by mismatching replicate numbers. The replicate correlation plot needs to normalize each selection replicate with its corresponding nonselect and wt replicates. for LDRL R2 the replicate numbers are:
```
 lowest  lowmid     mid midhigh highest     all      wt 
      2       2       3       2       3       2       2 
```
So, trying to normalize 3 `mid` replicates against 2 `all` and 2 `wt` replicates fails.

For now, I will just disable this subfunction when replicates don't match.

**Comment by jweile** [comment]
Fixed in 9656ec3
